.changeset/tough-tigers-drum.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': patch
+---
+
+Add default `.npmrc` file when adding the lit integration through `astro add lit` and using `pnpm`
```

Comment:
```suggestion
Add default `.npmrc` file when adding the Lit integration through `astro add lit` and using `pnpm`.
```
It looks like we capitalize Lit when we use the name in a sentence!

Reply:
Thanks! All the suggestions make sense to me